skipper.go

```diff
@@ -407,6 +409,12 @@ type Options struct {
 	// the last item of the string list of the X-Forwarded-For
 	// header, in this case you want to set this to true.
 	ReverseSourcePredicate bool
+
+	// TokenURL sets the TokenURL similar to https://godoc.org/golang.org/x/oauth2#Endpoint
```

Comment:
The endpoints in https://godoc.org/golang.org/x/oauth2#Endpoint are the authentication and token endpoints, but with this one you want the non-standard tokeninfo endpoint, right? What about servers without  a tokeninfo endpoint (e.g. https://github.com/coreos/dex/)

Reply:
I don't use this type anymore, thanks for pointing this out.